src/screens/administrationPanelView/administrationPanelViewStyles.ts

```diff
@@ -0,0 +1,43 @@
+import { styled } from '@mui/material';
+import { Avatar, Box, Button } from '@mui/material';
+
+import { theme } from '../../assets/styles/theme';
+
+export const Wrapper = styled(Box)`
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  width: 100%;
+  padding: 0 2rem;
```

Comment:
tutaj usuń ten padding bo te przyciski mają mieć 100% szerokości a ten padding nam to uniemożliwia